task-list-rails/app/views/tasks/show.html.erb

```diff
@@ -5,7 +5,7 @@
           <td colspan="2" id="title">Task details</td>
         </tr>
         <tr>
-          <td colspan="2" id=edit_link><%= link_to "edit?", "/tasks/#{@task.id}/edit" %></td>
+          <td colspan="2" id=edit_link><%= link_to "edit?", {controller: "tasks", action: "edit", id: @task.id} %></td>
```

Comment:
I like the use of the hash w/ controller, action, and params info here instead of a URL! I would recommend applying this same style throughout the rest of your view templates.